### ISSUE TITLE
Info: Does now openwrt support 802.11 r/k/v natively?

### ISSUE THREAD

**Issue by imwhocodes** [berlin-open-wireless-lab/DAWN#208]
Hi
I'm running openwrt snapshot and I noticed a new tab with new configuration about 802.11 r/k/v

It not completely clear what they do / how they integrate with other AP (**I'm** _supposing_ they work only for roaming from 2.4Gh to 5Gh on the same AP)

Is there any idea on how they should be configured to cooperate / integrate with DAWN?

`OpenWrt SNAPSHOT r21772-0a35d3f992 / LuCI Master git-22.361.69865-deed682`

![Screenshot 2023-01-13 at 00 34 22](https://user-images.githubusercontent.com/14922653/212204288-c5147e63-73fc-422d-890c-611ab05b8395.png)
![Screenshot 2023-01-13 at 00 34 31](https://user-images.githubusercontent.com/14922653/212204301-c08d42ef-feb0-4f22-b19f-07081c6859b8.png)

Thanks,
Luca

**Comment by rany2** [comment]
It does not support it without a third party program. Setting these only make it possible for an external program like DAWN to provide this information (or in the case of BSS Transition, take an action), but OpenWRT itself does not provide any functionality that would make these options useful.

**Comment by PolynomialDivision** [comment]
Indeed. There are some things you can do statically to make use of 802.11v, e.g. configuring neighbor reports that allows clients to skip scanning and directly connect to near aps.
